src/engraving/dom/chord.cpp

```diff
@@ -2538,6 +2542,75 @@ void Chord::toGraceAfter()
     }
 }
 
+bool Chord::isPreBendOrGraceBendStart() const
+{
+    if (!isGrace()) {
+        return false;
+    }
+
+    for (Note* note : m_notes) {
+        if (note->isPreBendStart() || note->isGraceBendStart()) {
+            return true;
+        }
+    }
+
+    return false;
+}
+
+bool Chord::preOrGraceBendSpacingExceptionInTab() const
```

Comment:
Could you please explain the purpose of this method?

Reply:
<img width="315" alt="image" src="https://github.com/musescore/MuseScore/assets/93707756/2b783a78-3b8a-4b28-8a0a-7210a06a6073">


Normally, grace notes are placed to the left of their main note (like in the normal staff here). In TABs however, because the main note isn't actually plucked when playing these bends, there are cases where the grace note should overlap the main note, which is why I need this spacing exception.